component/background/StorageService.tsx

```diff
@@ -13,9 +14,9 @@ class StorageService {
 
   /**
    * Clear the storage space and load with default settings
-   * @return {Promise<{}>} A promise whose resolve takes no argument, which runs after defaults are set
+   * @return {Promise<any>} A promise whose resolve takes no argument, which runs after defaults are set
    */
-  public reset(): Promise<{}> {
+  public reset(): Promise<any> {
```

Comment:
{} is object, it has no user-defined method. any can have anything on its interface, so {} is more type-safe